src/radar.cpp

```diff
@@ -191,6 +218,14 @@ static void radarSize(int ZoomLevel)
 		radarCenterY = pie_GetVideoBufferHeight() - BASE_GAP * 4 - static_cast<int>(radarHeight) / 2;
 	}
 	debug(LOG_WZ, "radar=(%u,%u) tex=(%zu,%zu) size=(%zu,%zu)", radarCenterX, radarCenterY, radarTexWidth, radarTexHeight, radarWidth, radarHeight);
+
+	if (pRadarWidget)
+	{
+		auto radarDiag = static_cast<size_t>(ceil(sqrt(double((radarWidth + 1) * (radarWidth + 1)) + double((radarHeight + 1) * (radarHeight + 1)))));
```

Comment:
Can be simplified:
```c++
size_t radarDiag = ceil(sqrt(...));
```

Reply:
Will cause compiler warnings, hence the explicit casting to make intent clear (and silence the warnings).